website/content/docs/concepts/host-discovery/aws.mdx

```diff
@@ -13,6 +13,119 @@ a dynamic host catalog to integrate with AWS, you create a host catalog of the `
 and set the `plugin-name` value to `aws`. You must also provide the specific
 fields needed for Boundary to authenticate with AWS.
 
+Boundary supports two methods of authenticating to AWS:
+
+1. **Static credentials** using an IAM user and its access key.
+1. **Dynamic credentials** using credentials generated by `AssumeRole`.
+
+HashiCorp recommends using dynamic credentials when possible. Select a credential type to continue.
+
+<Tabs>
+<Tab heading="Static credentials" group="static">
+
+### Required IAM Privileges
+
+Boundary needs the following IAM privileges, at the very least, to be attached to a configured IAM user.
+
+Configure `DescribeInstances` to `*`. `DescribeInstances` cannot be scoped to a resource ARN. 
+
+Example policy:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:DescribeInstances"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+To enable allow for static credential rotation, add the `iam:GetUser`, `iam:CreateAccessKey`, and `iam:DeleteAccessKey` policies:
```

Comment:
Should "allow" be formatted as `code` in this sentence?

Reply:
hmm I think I'll just rephase that as:

> To allow static credentials to be rotated, add the...

does that work?

Reply:
Yes, that sounds good to me! I misunderstood the sentence and was wondering if the allow was referring to the 
"Effect" in the code on line 42.